src/templates/haml/home3/index.html.haml

```diff
@@ -10,110 +10,88 @@
     .grd-container
       .grd-grid-12.grd-parent
         .zhead
-          %h1 The Premier Platform for Binary Options Trading
+          %h1= l('The Premier Platform for Binary Options Trading')
           .grd-grid-4.grd-grid-mobile-12.grd-parent
             %i.fa.fa-line-chart
           .grd-grid-4.grd-grid-mobile-12.grd-parent
-            %p!= "Up to <span> 100% </span> Return <br> on Rise/Fall Trades"
+            %p!= l('Up to <span> 100% </span> Return <br> on Rise/Fall Trades')
           .grd-grid-4.grd-grid-mobile-12.grd-parent
-            %p!= "Up to <span> 1000% </span> Return <br> on Trades you Make"
+            %p!= l('Up to <span> 1000% </span> Return <br> on Trades you Make')
         .zcta
           .button.pjaxload.zreal.pjaxload-focus.login-button
-            %span OPEN FREE ACCOUNT NOW
+            %span= l('Open Free Account Now')
 .grd-container
   #zsection1
     .grd-grid-12.grd-parent
-      %p.zheader.center The Easiest Way to Get Started in the Financial Markets
-      %p.center{:style => "font-size: 24px;"} Trade in the world's financial markets in a simple, user-friendly platform.
+      %p.zheader.center= l('The Easiest Way to Get Started in the Financial Markets')
+      %p.center{:style => "font-size: 24px;"}= l('Trade in the world\'s financial markets in a simple, user-friendly platform.')
       .grd-grid-7.grd-grid-mobile-12.grd-parent
-        %img.responsive{:alt => "", :src => url_for("images/pages/home3/trade-tutorial_1.svg")}/
+        %img.responsive{:alt => "", :src => url_for("images/pages/home3/trade-tutorial_1.svg")}
       .grd-grid-5.grd-grid-mobile-12.grd-parent
         %ul
-          %li
-            %i.fa.fa-check-square
-            %span All markets
-            \- currencies, stock indices, and commodities.
-          %li
-            %i.fa.fa-check-square
-            %span All market conditions
-            \- up/down, touch/no-touch, stays in/goes out.
-          %li
-            %i.fa.fa-check-square
-            %span All durations
-            \- from 10 seconds to 365 days.
-          %li
-            %i.fa.fa-check-square
-            %span All payouts
-            \- from $1 to $100,000.
+          %li!= l('<i class="fa fa-check-square"></i><span> All markets:</span> currencies, stock indices, and commodities.')
+          %li!= l('<i class="fa fa-check-square"></i><span> All market conditions:</span> up/down, touch/no-touch, stays in/goes out.')
+          %li!= l('<i class="fa fa-check-square"></i><span> All durations:</span> from 10 seconds to 365 days.')
+          %li!= l('<i class="fa fa-check-square"></i><span> All payouts:</span> from $1 to $100,000.')
         .center{:style => "margin-top:40px;"}
-          %a#signbox.button.pjaxload.pjaxload-focus{:href => "https://www.binary.com/c/linkto_acopening.cgi?actype=real&l=EN"}
-            %span OPEN A REAL MONEY ACCOUNT
+          %a#signbox.button.pjaxload.pjaxload-focus{:href => url_for('open_account.cgi')}
```

Comment:
Instead of open_account.cgi we need to give the url to open account 
https://www.binary.com/c/linkto_acopening.cgi